build/dependencies.props

```diff
@@ -391,7 +392,7 @@ not building again in this patch.
     <ExternalDependency Include="Microsoft.AspNetCore.Rewrite" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
     <ExternalDependency Include="Microsoft.AspNetCore.Routing.Abstractions" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
     <ExternalDependency Include="Microsoft.AspNetCore.Routing" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
-    <ExternalDependency Include="Microsoft.AspNetCore.Server.HttpSys" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
+    <ExternalDependency Include="Microsoft.AspNetCore.Server.HttpSys" Version="2.0.2" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
```

Comment:
Not sure why this change is just now happening -- it's 2.0.2 on nuget for a while.

Reply:
2.0.2 was released out-of-band from the metapackage. We didn't have proper follow-thru to update the metapackage in the subsequent release. I just noticed now as I was scanning for latest package versions.